tasks/test.py

```diff
@@ -132,7 +132,7 @@ def invoke_unit_tests(ctx):
     for _, _, files in os.walk("tasks/unit-tests/"):
         for file in files:
             if file[-3:] == ".py" and file != "__init__.py" and not bool(UNIT_TEST_FILE_FORMAT.search(file[:-3])):
-                ctx.run(f"python3 -m tasks.unit-tests.{file[:-3]}")
+                ctx.run(f"python3 -m tasks.unit-tests.{file[:-3]}", env={"GITLAB_TOKEN": "fake_token"})
```

Comment:
Why did you have to add that to the env var? Wasn't it working before?

Reply:
Yep it was working before because, no gitlab tests existed. Since I added some gitlab tests they're looking for the ``GITLAB_TOKEN`` environment variable to get the token. Since most runners don't have this var I need to add it in the context, since the requests get mocked after. I don't need a real token but I need one to be found otherwise it will crash.